PlayerInventoryUI.gd

```diff
@@ -1,81 +1,85 @@
 extends Control
 
-var item_count = 0
-var grid_pos = 0
-var cursor_pos = 0
+@onready var faux_item_collection = get_tree().get_first_node_in_group("FauxColl")
+@onready var item_collection = get_tree().get_first_node_in_group("ItemColl")
+@export var grav_affected_item_scene: PackedScene
 
-func add_item(item):
-	
-	item.reparent(get_child(grid_pos).get_child(item_count))
-	item.position = Vector2(26,39) 
-	item_count += 1
-	item.get_child(0).flip_v = false
-	item.rotation_degrees = 0
-	sort()
-
-func delete_item(pos):
-	get_child(grid_pos).get_child(pos).get_child(0).queue_free()
-	item_count -= 1
-	sort()
+var maxInCollection = 10
```

Comment:
Not sure if this is the right approach anymore

Reply:
https://github.com/StarkandcoEnterprises/Secret-game/pull/55/commits/b528671152748930312360d5526b6ed4517f5b04